src/test/java/com/icthh/xm/ms/mstemplate/config/AsyncSyncConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package com.icthh.xm.ms.mstemplate.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.task.SyncTaskExecutor;
+
+import java.util.concurrent.Executor;
+
+@Configuration
+public class AsyncSyncConfiguration {
```

Comment:
what actually do this class? why it need?

Reply:
Creates taskExecutor with SyncTaskExecutor for test purposes. No need to do waits in tests.